tests/angular/app/components/login-form/login-form.component.spec.js

```diff
@@ -0,0 +1,23 @@
+ngDescribe({
+    name: 'Test login-form component',
+    modules: 'app',
+    tests: function ($rootScope, $compile) {
```

Comment:
An easier way to do this is by just passing an `element` field
`element: '<login-form></login-form>`
one of the main reasons why this is so convenient :smile: 
Take a look under the [primary options](https://github.com/kensho/ng-describe#primary-options) for the `element` property